src/api/taxes.js

```diff
@@ -0,0 +1,15 @@
+const fetchTaxes = async (csrf, apiPath) => {
+  const response = await fetch(`${apiPath}/taxes`, {
+    mode: 'cors',
+    method: 'POST',
```

Comment:
That's what I originally though too. But it's a post because it's actually generating the taxes on the server when you hit this endpoint and then returning them. https://developer.boldcommerce.com/default/documentation/checkout#/Taxes/post-taxes